app/controllers/users_controller.rb

```diff
@@ -1,30 +1,12 @@
 class UsersController < ApplicationController
-  before_action :logged_in_user, only: [:edit, :index, :update, :destroy]
-  before_action :correct_user, only: [:edit, :update]
-  before_action :admin_user, only: :destroy
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user,   only: [:edit, :update]
+  before_action :admin_user,     only: :destroy
```

Comment:
Indent error. Should set 1 space
Example: before_action :admin_user, only: :destroy